tests/users/test_api_update_user.py

```diff
@@ -37,121 +32,107 @@ def test_update_username_already_taken(self):
         db.session.add(self.first_user)
         db.session.commit()
 
+    def test_update_user_api_resource_non_auth(self):
```

Comment:
Can you just add only the tests and leave the pep8 issues and other code changes for now? In opensource it is considered a good practice to only fix one issue per pull request.Thanks :)